tests/istanbul/eip152_blake2/test_blake2.py

```diff
@@ -0,0 +1,501 @@
+                }
+
+
+                // This case is provided by jochem-brouwer in 
+                // https://github.com/ethereum/tests/pull/948#issuecomment-925964632  
+                case 9 {
+                    mstore(0x00, 0x0000000c48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f)
+                    mstore(0x20, 0x3af54fa5d182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e13)
+                    mstore(0x40, 0x19cde05b61626364650000000000000000000000000000000000000000000000)
+                    mstore(0x60, 0x0000000000000000000000000000000000000000000000000000000000000000)
+                    mstore(0x80, 0x0000000000000000000000000000000000000000000000000000000000000000)
+                    mstore(0xa0, 0x0000000000000000000000000000000000000000000000000000000000000000)
+                    mstore(0xc0, 0x0000000005000000000000000000000000000000010000000000000000000000)
+                }
+
+                // if param is over 0x0F then it's the number of rounds
+                if gt(param, 0x0F) {
+                    mstore(0x00, 0x0000000048c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f)
+                    mstore(0x20, 0x3af54fa5d182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e13)
+                    mstore(0x40, 0x19cde05b61626300000000000000000000000000000000000000000000000000)
+                    mstore(0x60, 0x0000000000000000000000000000000000000000000000000000000000000000)
+                    mstore(0x80, 0x0000000000000000000000000000000000000000000000000000000000000000)
+                    mstore(0xa0, 0x0000000000000000000000000000000000000000000000000000000000000000)
+                    mstore(0xc0, 0x0000000003000000000000000000000000000000010000000000000000000000)
+
+                    // The first four bytes are the number of rounds.
+                    mstore8(0x00, shr(24, and(param, 0xFF000000)))
+                    mstore8(0x01, shr(16, and(param, 0x00FF0000)))
+                    mstore8(0x02, shr( 8, and(param, 0x0000FF00)))
+                    mstore8(0x03,         and(param, 0x000000FF))
+                }  // if param is over 0x0F
+
+                // The message starts at byte 0x44
+                if gt(msgLen, 0) {
+                    for { } gt(msgLen, 0) { } {
+                        // Write a byte
+                        mstore8(add(0x44, msgLen), add(0x60, msgLen))
+                        msgLen := sub(msgLen, 1)
+                    }
+                }
+
+                // The input is bytes 0-bufLength
+                // The output is bytes 0x200-0x23F
+                let retVal := call(gas(), 9, 0, 0, bufLength, 0x200, 0x40)
+
+                // Return the results
+                sstore(0, retVal)   // Success? Failure?
+                // The output
+                sstore(1, mload(0x200))
+                sstore(2, mload(0x220))
+            }
+            """
```

Comment:
This will allow lines > 100 in the Yul block for convenience :)
```suggestion
            """  # noqa: E501 (line-too-long)
```